src/schema/me/__tests__/bidder_positions.test.js

```diff
@@ -1,99 +1,99 @@
-import { map, times } from "lodash"
+import { map } from "lodash"
 import gql from "test/gql"
 
-import schema from "schema"
 import { runAuthenticatedQuery } from "test/utils"
 
 describe("Me type", () => {
-  const Me = schema.__get__("Me")
-  const BidderPositions = Me.__get__("BidderPositions")
-  const BidderPosition = BidderPositions.__get__("BidderPosition")
-
-  let gravity
-  let gravity2
-
+  let rootValue
+  let saleArtworkRootLoader
   beforeEach(() => {
-    gravity = sinon.stub()
-    gravity2 = sinon.stub()
-    gravity.with = sinon.stub().returns(gravity)
+    saleArtworkRootLoader = jest.fn()
+    saleArtworkRootLoader
```

Comment:
I don’t _think_ you need to do this on another line, but chain it to `jest.fn()` instead.